### PR TITLE
Simplify setting up the space with a little script.

### PR DESCRIPTION
**Pull request by realityking**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,9 @@ Super Chemex Bot is only a small piece of code but requires a bit of configurati
 
  * Clone this repository
  * `cd` into the `lambda` directory and run `npm install` (`yarn install` is also supported).
- * Set up a Contentful space.
-   * You'll need to create two content types
-     * One with the ID `configuration` and two short text fields with the IDs `deviceSerialNumber` and `slackChannel`
-     * The other with the ID `message` and a short text field with the ID `message`
-   * Create at least one entry for each of these content types. One mapping a button serial number to a slack channel, the other with the message you want to post.
-   * Create an API key. You'll need it later.
+ * Sign-up for Contentful if you haven't already.
```
Comment (stefanjudis) [inline README.md]: It  should be `Sign up`

File: setup-space.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+if (!program.managementToken) {
+  console.log('You need to specify your management token using "--management-token"');
+  process.exit();
+}
+
+const accessToken = program.managementToken;
+const organizationId = program.organizationId ? program.organizationId : undefined;
+const spaceId = program.spaceId ? program.spaceId : undefined;
+
+const spaceData = JSON.parse(fs.readFileSync('./space.json'));
```
Comment (stefanjudis) [inline setup-space.js]: `require` would work just fine here or? :)

**Review by stefanjudis — CHANGES_REQUESTED** [review]
Tiny nitpicks. :)

**Comment by realityking** [comment]
@stefanjudis All adressed, please have another look.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "license": "MIT",
+  "private": true,
+  "dependencies": {
+    "commander": "^2.15.1",
+    "contentful-import": "^7.0.1",
+    "contentful-management": "^5.0.0",
+    "pify": "^3.0.0"
+  }
```
Comment (stefanjudis) [inline package.json]: You're not using this one or? :)

**Review by stefanjudis — CHANGES_REQUESTED** [review]
Tiny nit... apart from that 👌🏻

File: package.json (hunk 3 above)
Comment (stefanjudis) [reply package.json]: `pify` i mean. :)

**Pull request merged**